lib/tasks/indices.rake

```diff
@@ -186,6 +186,15 @@ this task will run against all active clusters.
     end
   end
 
+  desc "Cleans out unused indices older than the number of days given by the MAX_INDEX_AGE flag"
+  task :timed_clean do
+    Clusters.active.each do |cluster|
+      index_names.each do |index_name|
+        search_server(cluster: cluster).index_group(index_name).timed_clean(max_index_age)
```

Comment:
Does this work?  I'm wondering how `max_index_age` is assigned

Reply:
It's a flag the same way SEARCH_INDEX is. So example usage:

`rake search:timed_clean SEARCH_INDEX=all MAX_INDEX_AGE=7`

Works as designed locally, complete with yelling at you if you don't use that flag. 👍